nucliadb/nucliadb/ingest/orm/processor/__init__.py

```diff
@@ -515,6 +516,19 @@ async def maybe_update_resource_basic(
             deleted_fields=message.delete_fields,  # type: ignore
         )
 
+    async def get_extended_audit_data(
+        self, message: writer_pb2.BrokerMessage
+    ) -> writer_pb2.Audit:
+        message_audit = writer_pb2.Audit()
+        message_audit.CopyFrom(message.audit)
+        message_audit.kbid = message.kbid
+        message_audit.uuid = message.uuid
+        message_audit.message_source = message.source
+        message_audit.field_metadata.extend([fi.field for fi in message.field_metadata])
```

Comment:
Isn't the field metadata still too big to send in every audit?

Reply:
We're only sending the field ids though